src/components/Other/navbar.js

```diff
@@ -24,51 +22,50 @@ export default function Navbar () {
     }
   }
 
+  const isDesktop = useMediaQuery({ query: '(min-width: 768px)' })
   window.addEventListener('scroll', setDropShadow)
 
   return (
     <nav className={navbar ? 'navbar active' : 'navbar'}>
       <div className='navbar-content'>
         {isDesktop && <div className='desktop-content'>
 
-          <Link
-            className='sanctuary-logo'
-            to='/'
-          >
+          <Link className='sanctuary-logo' to='/'>
             <SanctuaryLogo />
           </Link>
 
           <div className='navbar-links'>
             <Search onClick={() => dispatch(showSearchModal())} />
-            <Link
-              onClick={() => document.location.href = '/browse'}
-            >
+            <Link onClick={() => { document.location.href = '/browse' }} to='/browse'>
               Browse
             </Link>
 
-            <Link
-              onClick={() => document.location.href = '/newsroom'}
-            >
+            <Link onClick={() => { document.location.href = '/portfolio' }} to='/portfolio'>
+              Portfolio
+            </Link>
+
+            <Link onClick={() => { document.location.href = '/newsroom' }} to='/newsroom'>
               Newsroom
             </Link>
-            {!user &&
-              <div className="login" onClick={() => { document.location.href = '/sign-in-email' }}>Log in</div>
-            }
+
+            {/* {!user &&
```

Comment:
[BLOCKING] This should be uncommented right?

Reply:
this should be deleted actually, we don't want an extra sign out button in the navbar

Reply:
we actually don't need two buttons for auth b/c the sign up page is accessible from the login page (usually how it's done in most websites for simplicity). I can remove the commented code tho